src/dataconnect/provisionCloudSql.ts

```diff
@@ -95,23 +106,23 @@ export async function provisionCloudSql(args: {
 export function checkInstanceConfig(
```

Comment:
nit: Maybe rename to `getUpdateReason` and have this return an array of string reasons?

Reply:
Done